src/bin/common.rs

```diff
@@ -84,6 +85,14 @@ pub fn parse_cli() -> CliOptions {
         .takes_value(true)
         .default_value("0")
     )
+    .arg(
+      Arg::with_name("SKIP")
+        .help("Skip n number of frames and encode")
+        .short("s")
```

Comment:
The "s" short option is already used by speed, so this should either be omitted or something other than "s".

Reply:
We can discard it as --skip makes more sense to me :D